.github/workflows/python-build-package.yml

```diff
@@ -60,6 +60,19 @@ jobs:
       - if: matrix.platform.runner != 'windows-latest'
         name: Check that `uv add magika.whl` works
         run: mkdir /tmp/test-uv && cp -vR dist/*.whl /tmp/test-uv && cd /tmp/test-uv && uv init && uv add ./$(\ls -1 *.whl | head -n 1)
+      - if: matrix.platform.runner == 'windows-latest'
+        name: Install uv on Windows
```

Comment:
do we really need this? Line 37 should already install uv on windows as well? (same for the other two files)

Reply:
As I understand it, line 37 uses a shell script which will not work on windows. That's why I added a separate step that uses a powershell script to handle uv install.

Reply:
From https://github.com/google/magika/actions/runs/12232019796/job/34140885923, it seems that it does run successfully (the command is run for sure as there is no `if` on platform runner OS; question is whether your powershell part can use the just-installed uv). The problematic part was the one-liner starting with `mkdir`, which your PR seems to address. Can you maybe try to remove the "install uv on windows" part and check that it still works?

Reply:
Done! I removed that step and it still works fine